ui/app/components/ui/alert/index.js

```diff
@@ -45,9 +40,9 @@ class Alert extends Component {
   render () {
     if (this.state.visible) {
       return (
-        h(`div.global-alert${this.state.className}`, {},
-          h('a.msg', {}, this.state.msg)
-        )
+        <div className={classnames('global-alert', this.state.className)}>
```

Comment:
The two class names that could be here (`.visible` and `.hidden`) _start_ with a period because they were intended to be embedded in that hyperscript string. The leading period should be removed now that they're set like this.

Also maybe they should be set like this?
```
classnames('global-alert', {
  [this.state.className]: this.state.className
})
```
So that we don't end up setting an empty string class if `state.className` is empty.

Reply:
Thanks, this is a good catch

Reply:
Fixed the leading period. `classnames` will ignore the empty string since "falsy values are just ignored".